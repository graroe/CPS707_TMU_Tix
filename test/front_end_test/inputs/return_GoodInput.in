login
sales
return
CirqueDuSoleil
6
return
CirqueDuSoleil
2
logout