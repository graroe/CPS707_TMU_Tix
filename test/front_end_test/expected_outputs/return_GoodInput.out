Enter session type <sales or admin>: 
Sales login successful.
Enter event name: 
Current number of tickets: 10\nEnter number of tickets to return: 
6 tickets returned. New total: 16
Enter event name: 
Current number of tickets: 16\nEnter number of tickets to return: 
2 tickets returned. New total: 18
Session terminated.